tox.ini

```diff
@@ -7,6 +7,7 @@ sitepackages = True
 passenv = HOME CHIRP_TESTS CHIRP_TESTIMG
 whitelist_externals = bash
 deps = future
+use_develop = True
```

Comment:
Why are you making this change?

Reply:
`tests/` and `tools/` are not included in bdists and binary installations, which `tox` does by default otherwise (default `install_command` is `python -m pip install {opts} {packages}`)

Reply:
`serial` refers to a completely unrelated PyPI package, the correct specification is `pyserial`.

Reply:
I know what develop does, I'm just wondering why you're changing this. Even with the sdist install, we're running the tests locally and not from the installed base, yeah? I know develop makes it a little faster (which is why I was doing it before) but it's not really measurable locally and it can lead to the tests being able to import stuff that didn't get into the full build.

Reply:
> `serial` refers to a completely unrelated PyPI package, the correct specification is `pyserial`.

I know, i constantly make this mistake. Even after 15 years.